style.css

```diff
@@ -0,0 +1,69 @@
+* {
+  box-sizing: border-box;
+  margin: 0 2px;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style-type: none;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px 0;
+}
+
+.header a {
+  color: #6070ff;
+}
+
+.header-brand {
+  font-weight: 700px;
+  font-size: 25px;
+  color: #28352f;
+}
+
+.header-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(./img/header-bg-mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
Kindly add some padding to this section to make it look more similar to the Figma template :wink: